src/pages/data-science-tools/index.md

```diff
@@ -8,3 +8,5 @@ In this section, we'll have guides to a wide variety of tools used by data scien
 Data scientists are inquisitive and often seek out new tools that help them find answers. They also need to be proficient in using the tools of the trade, even though there are dozens upon dozens of them. Overall, data scientists should have a working knowledge of statistical programming languages for constructing data processing systems, databases, and visualization tools. Many in the field also deem a knowledge of programming an integral part of data science; however, not all data scientist students study programming, so it is helpful to be aware of tools that circumvent programming and include a user-friendly graphical interface so that data scientists’ knowledge of algorithms is enough to help them build predictive models.
 
 R remains the leading tool, with 49% share, but Python grows faster and almost catches up to R. RapidMiner remains the most popular general Data Science platform. Big Data tools used by almost 40%, and Deep Learning usage doubles.
+Data Science is OSEMN O= Obtain , S= Scrub,M=Model, N= iNterpret the Data.
+There is one good resource for Data Science and Machine Learning by Open Source Data Science Masters.Follow on github  datasciencemasters!!!*[Resouses for Data Science](https://github.com/datasciencemasters/go)
```

Comment:
1 Indentation before "Follow" as well.
Also, you have a typo in "Resouses". Move that to a new line aswell. Thanks.